backend/open_webui/utils/auth.py

```
@@ -167,3 +169,31 @@ def get_admin_user(user=Depends(get_current_user)):
+    headers = {
+        "Authorization": f"Bearer {LITELLM_MASTER_KEY}",
+        "Content-Type": "application/json"
+    }
+
+    # TODO: Other things can be setted up, depending on the offer & pricing: Duration, etc. We can define a Pydantic Model for each offer for better modeling
+    data = {
+        "metadata": {"user": f"{user_name}"},
+        'max_budget': 10,
+        # 'budget_duration': '1h',
+        'key_alias': f"{user_name}",
+        'metadata': {"team": "AiTeam", "user":user_name, "email": email}
```

Ruff (0.8.2)

Dictionary key literal `'metadata'` repeated (`'metadata'` hashes to the same value as `"metadata"`)

(F601)